conda.recipe/meta.yaml

```diff
@@ -0,0 +1,76 @@
+package:
+  name: holoplot
+  version: 0.0.1
+
+source:
+  path: ..
+
+build:
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+extra:
+  channels:
+    - pyviz
+    - intake
+
+requirements:
+  build:
+    - python
+    - param
+    - numpy
+    - setuptools
+  run:
+    # 
+    # Versions for pyviz-maintained packages pinned to ensure mutual compatibility
+    # 
+    - bokeh 0.12.14
+    - holoviews 1.9.5
+    - datashader 0.6.5
+    - param 1.5.1
+    # 
+    # Direct dependencies for package installation
+    # 
+    - yaml
+    - requests
+    # 
+    # Direct dependencies of most or all the tutorials
+    # 
+    - python >=3.6,<3.7 # Pinned in case of unknown future or past compatibility issues
+    - ipython 5         # Pinned to avoid tab-completion issue in ipython=6
+    - numpy 1.13        # Pinned to avoid warnings with numpy=1.14 and bokeh=1.12.14
+    - notebook >=5.1    # Pinned to avoid rate limiting for large images with notebook=5.0
+    - dask 0.17.1       # Pinned in case of future API changes
+    - jupyter
+    - pandas
+    - xarray
+    - scipy
+    # 
+    # Directly imported dependencies of certain specialized tutorials
+    # 
+    - streamz 0.3.0 
+    # 
+    # Indirect dependencies of our direct dependencies
+    # 
+    - selenium          # Required for exporting Bokeh plots to PNG
+    - phantomjs         # Required for exporting Bokeh plots to PNG
+    - netcdf4           # Required for reading .nc files
+    - fastparquet       # Required for reading Parquet files
+    - python-snappy     # Required for reading snappy-compressed Parquet files
+    - pytest >=3.2      # Required by? Pinned to avoid installation problems on Windows with 3.1?
+    - cffi              # Unstated dependency of fastparquet?
+    - tornado 4.5.3     # Required for Jupyter notebook and bokeh server
+    # Intake data packages used in documentation #
+    - airline_flights
+    - us_crime
+    - intake
+    - intake-parquet
+
```

Comment:
Why are xarray, intake, dask, and pandas in the run requirements (i.e., dependencies)?  I thought the point of this package was that it could be installed alongside any of those packages, without requiring any of the others.  Shouldn't those be in the test requirements instead?

Reply:
Agreed, I'll have to update this. I started by copying the pyviz meta.yaml but need to pare it down now.

Reply:
Ok, good.